ordering/services/orders.js

```diff
@@ -28,6 +32,38 @@ const hooks = {
   error: {}
 }
 
+function getCurrentUser (hook) {
+ const { agentId } = hook.params.credential
+ const agentsService = hook.app.service('agents')
+ return agentsService.get(agentId)
+ .then(agent => {
+   hook.params.agent = agent
+ })
+ .then(() => hook)
+}
+
+function groupHasNoAdminRelation(hook) {
+  const relationships = hook.app.service('relationships')
+  const groupId = hook.data.consumerAgentId
+  return relationships.find({ query: {targetId: groupId} }).then((relationship)=>{
```

Comment:
could be fancy and pass in `isEmpty` directly to `.then()`, otherwise it's `relationships` yeah?